weekly-updates/weekly-update.2015-06-19.md

```diff
@@ -0,0 +1,47 @@
+# io.js News
```

Comment:
We are going to publish it on io.js Medium, so.... I think it is still io.js for now. But I you think we should change it, I don't have problem with it.